Gemfile

```diff
@@ -61,7 +61,7 @@ gem 'kaminari', '~> 1.2'
 gem 'link_header', '~> 0.0'
 gem 'mime-types', '~> 3.4.1', require: 'mime/types/columnar'
 gem 'nokogiri', '~> 1.14'
-gem 'nsa', '~> 0.2'
+# gem 'nsa', '~> 0.2'
```

Comment:
This gem looks not maintained ... the last changes were from you almost 3 years ago working on rails 6 support.

Should we do that again? Are there alternate gems or other logging approaches which could fill the same need?